site/src/nav/DownloadButton.css

```diff
@@ -59,3 +59,70 @@ svg.icon-refresh > path.secondary {
 div.download-text {
   padding-left: 1px;
 }
+
+div.filename-field-wrapper {
+  background: white;
+  position: absolute;
+  padding-right: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border: solid 1px black;
+  border-radius: 5px;
+  width: 280px;
+  margin-left: -83.33px;
+  margin-top: 12px;
+}
+
+div.ffw-dark {
+  background: dimgray;
```

Comment:
Can we use the Infima variable for the navbar here?

This grey is lighter and provides less contrast for dark mode. Using shared CSS variables as much as possible will lead to a more consistent UI.

Reply:
I can, I think I was more hesitant too match the navbar color completely. I wasn't 100% if the style we wanted to use for dark mode was to match the navbar, so I decided to make it a noticeably different color to differentiate it. I will change it to match now.